test_runner/src/main/kotlin/ftl/run/DumpShards.kt

```diff
@@ -32,10 +33,22 @@ fun IosArgs.dumpShards(
     // VisibleForTesting
     shardFilePath: String = IOS_SHARD_FILE,
 ) {
+    val xcTestRunShards: Map<String, List<List<String>>> = xcTestRunData.shardTargets.mapValues {
+        it.value.flatMap { it.values }
+    }
+
+    val rawShards: Any = when (xcTestRunData.version) {
+        V1 -> xcTestRunShards.values.first()
+        V2 -> xcTestRunShards
+    }
+
+    val size = xcTestRunData.shardTargets.values
+        .flatten().size
```

Comment:
Is this correct? I'm not sure if `size` will be correct for xctestrun V1. 
Should we return `rawShards.size` here?